File: monitor/v1/event/read-example-sourceipaddress-filter/read-example-sourceipaddress-filter.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Monitor.V1;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var events = EventResource.Read(
            endDate: new DateTime(2015, 4, 25, 0, 0, 0),
            sourceIpAddress: "104.14.155.29",
            startDate: new DateTime(2015, 4, 25, 0, 0, 0)
        );

        foreach(var record in events)
        {
           Console.WriteLine(record.Sid);
        }
    }
}

File: monitor/v1/event/read-example-sourceipaddress-filter/read-example-sourceipaddress-filter.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$events = $twilio->monitor->v1->events
                              ->read(array(
                                         "endDate" => new \DateTime('2015-4-25'),
                                         "sourceIpAddress" => "104.14.155.29",
                                         "startDate" => new \DateTime('2015-4-25')
                                     )
                              );

foreach ($events as $record) {
    print($record->sid);
}